### PR TITLE
feat(vfg): convert field-radio [khcp-11342]

### PR DESCRIPTION
**Pull request by kaiarrowood**
# Summary

Convert `FieldRadio` to use kongponents for [KHCP-11342](https://konghq.atlassian.net/browse/KHCP-11342).

Adoption with passing E2Es: https://github.com/Kong/konnect-ui-apps/pull/3958

#### Resources

- [Creating a package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md)

[KHCP-11342]: https://konghq.atlassian.net/browse/KHCP-11342?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/forms/src/components/fields/FieldRadio.vue [hunk 1]
```diff
@@ -1,72 +1,97 @@
...
-            :value="option.value"
-            @change="onChange"
-          >
-          {{ option.name }}
-        </label>
-      </div>
-    </div>
+      :id="schema.name+'-'+i"
+      :key="option.value"
+      v-model="inputValue"
```
Comment (sumimakito) [inline packages/core/forms/src/components/fields/FieldRadio.vue:7]: Although the `KRadio` worked as expected when we passed `null` to `model-value`, I saw some warnings thrown from it, does it officially support setting the `undefined`/`null` as `model-value`?

```
[Vue warn]: Invalid prop: type check failed for prop "modelValue". Expected String | Number | Boolean | Object, got Null  
  at <KRadio id="gender-0" key="male" modelValue=null  ... > 
  at <FieldRadio ref="child" disabled=null form-options=  ...
  ...
```

**Review by sumimakito — APPROVED** [review]
Did functionality test and LGTM. Left a non-blocking comment

File: packages/core/forms/src/components/fields/FieldRadio.vue (hunk 1 above)
Comment (kaiarrowood) [reply packages/core/forms/src/components/fields/FieldRadio.vue:7]: Addressing in https://github.com/Kong/kongponents/pull/2342
I've also added a test in FieldRadio to verify unsetting works 👍🏻

Comment (kaiarrowood) [reply packages/core/forms/src/components/fields/FieldRadio.vue:7]: Change has been picked up in the repo

**Review by sumimakito — APPROVED** [review]
LGTM

**Pull request merged**